sound/ambience/maintenance/attribution.txt

```diff
@@ -1,2 +1 @@
-ambispace4.ogg is Sipping Bird, by Capsandi. It has been licensed under CC-BY 3.0 license.
-ambimaint6.ogg and ambimaint7.ogg, created by Capsandi are licensed under CC-BY 3.0 license.
+magma.ogg - Hot spring.Seething and bubbles(2lrs,mltprcssng).wav by newlocknew -- https://freesound.org/s/581417/ -- License: Attribution 4.0 and wind 5 by ZIP.Creates -- https://freesound.org/s/726316/ -- License: Creative Commons 0
```

Comment:
im sure you've explained somewhere but can you explain on github for looking back in the future why you're removing attribution for ambispace4 and ambimaint6?
Also,
1. ``magma.ogg`` is not in this folder
2. this seems like a copy paste of ``sound/ambience/attribution.txt``
3. What's 'wind 5'? I can't find a file called that, but listening in to both sounds individually and the magma sound we have in game, sounds like you merged the two into one sound, can you specify that it is the case, in the proper file, as per the requirements

![image](https://github.com/user-attachments/assets/a9281dd6-09df-4257-ba10-037108966548)

Reply:
mistakes were made, I'll fix it. the license fix should probably be in anothe pr tho